configs/base/packages.yaml

```diff
@@ -18,7 +18,7 @@ packages:
   tioga:
     version: [develop]
   openfast:
-    version: [develop]
```

Comment:
Does 3.5.3 have the fsi features?  I was under the impression they wouldn't be coming until 4.0.

Reply:
oh maybe not

Reply:
I will just close this. I think you're right.